lib/coophub/cache_warmer.ex

```diff
@@ -42,11 +42,16 @@ defmodule Coophub.CacheWarmer do
 
     repos =
       read_yml()
-      |> Enum.reduce([], fn {key, yml_data}, acc ->
-        case get_org(key, yml_data) do
-          :error -> acc
-          org -> [get_repos(org) | acc]
-        end
+      |> Enum.map(fn {key, yml_data} ->
+        Task.async(fn ->
+          case get_org(key, yml_data) do
+            :error -> []
```

Comment:
What happens if getting an org fails? That empty list should be considered as "no organization".
Shouldn't be `nil` or something like that? Because we aren't accumulating as before with `Enum.reduce/`, this empty list may break things up!
The original behaviour was to **remove** failed orgs from results (and should be the same when getting repos).

Reply:
that part was not modified

Reply:
Added `List.flatten/1` to remove failed `[]` cases!